src/v2/BaseLoginRouter.js

```diff
@@ -130,6 +142,50 @@ export default Router.extend({
     }
   },
 
+  handleTerminalStateWithNoRemediation () {
+    const deviceEnrollmentType = this.settings.get('deviceEnrollment.name');
+    let deviceEnrollmentResponse;
+
+    if (deviceEnrollmentType === 'oda') {
+      deviceEnrollmentResponse = {
```

Comment:
remove it from here and pass as options in ``deviceEnrollmentTerminalView` function in WidgetRouter.js